### ISSUE TITLE
Onion Peel on transformed rectilinear grid crashes parallel engine

### ISSUE THREAD

**Issue by aowen87** [aowen87/FOO#213]
Open chgcar_test_data/CHGCAR
Add PC plot
Apply OnionPeel operator 
Draw

parallel engine will crash.

Seems in part due to OnionPeel's use of vtkDataSetRemoveGhostCells.
commenting-out use of the filter allows OnionPeel to complete, but gives different
results than when run in serial.

Restructuring the code a bit:
old:
   ds = removeGhostCells->GetOutput();
   ds->Update();
new:
   removeGhostCells->Update();
   ds = removeGhostCells->GetOutput();

also removes the crash, but OnionPeel can fail with bad index.
eg. try seed cell of 97, which works in serial but in parallel
yields bad index error saying max is 49.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Kathleen Biagas
Original creation: 02/24/2011 12:42 pm
Original update: 02/24/2011 03:17 pm
Ticket number: 632
